### PR TITLE
Review contributing

### PR DESCRIPTION
**Pull request by cpholguera**
- Increase readability and scannability 
- Fixed minor typos/format issues
- Added reference to Code of Conduct

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 40eb9a590380ab24282bb79cb7b2e6c46557e6c7: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,23 +4,23 @@
...
+- **File [issues](https://github.com/OWASP/owasp-mstg/issues "MSTG Issues")** for missing content or errors. Explain what you think is missing and give a suggestion as to where it could be added.
+- **Create a [Pull Request (PR)](https://github.com/OWASP/owasp-mstg/pulls "Create a pull request")**. This is a direct contribution to the guide and may be merged after review. Be sure to follow our [style guide](https://github.com/OWASP/owasp-mstg/blob/master/style_guide.md "MSTG Style Guide") when writing content. You should ideally [create an issue](https://github.com/OWASP/owasp-mstg/issues "MSTG Issues") for any PR you would like to submit, as we can first review the merit of the PR and avoid any unnecessary work. This is of course not needed for small modifications such as correcting typos.
+- **Review pull requests**. If you are a fluent speaker in any of the different languages that the MSTG is available in, feel free to give feedback on any of the submitted PRs.
```
Comment (commjoen) [inline CONTRIBUTING.md]:
```suggestion
- **Review pull requests for style, spelling and grammer**. If you are a fluent speaker in any of the different languages that the MSTG is available in, feel free to give feedback on any of the submitted PRs.
- **Review pull requests for technical content**. Do you have tons of experience with the technical subject described in the PR? Chip in with constructive feedback.
```

**Comment by OMTGreleaser** [comment]
Results for commit 16227549c4aa911dcc8d0334e4b402bd9ca996b7: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by commjoen** [comment]
Thanks!

**Pull request merged**